src/Nethermind/Nethermind.TxPool/TxBroadcaster.cs

```diff
@@ -100,7 +101,7 @@ private void StartBroadcast(Transaction tx)
             NotifyPeersAboutLocalTx(tx);
             if (tx.Hash is not null)
             {
-                _persistentTxs.TryInsert(tx.Hash, tx);
+                _persistentTxs.TryInsert(tx.Hash, tx.SupportsBlobs ? new LightTransaction(tx) : tx);
```

Comment:
Wouldn't/Shouldn't LightTransaction be already passed here?

Reply:
Then this logic would need to be moved to TxPool AddCore - I think it is better to have it in TxBroadcaster, as low as possible